### ISSUE TITLE
Unable to start substitute on A12 XS 12.4 

### ISSUE THREAD

**Issue by sooxt98** [pwn20wndstuff/Undecimus#1514]
![IMG_5950](https://user-images.githubusercontent.com/13378059/65417945-d6ca7d80-de2d-11e9-835d-4416ee355a3e.PNG)

settings enabled: Load tweak & load daemons only

**Comment by elwoodjd** [comment]
Same issue now - the first run through I was able to jailbreak the device.  Cydia showed up.  I could open Cydia, and I tried to download and install Apex2 (a tweak I was able to run when I was running 12.1.2 on my XS with the Chimera break some time ago).

After Apex2 installed and my device resprung, I could do get Cydia to open.  I would open Uncover and it would re-jailbreak in about 4 seconds.  I deleted uncover and re-sideloaded it, ran it again.  Seemed to work until it said it couldn't start substitute. 

I resprung, now Cydia crashes on open, Uncover crashes on open, and my volume/power buttons are acting weird (I cannot turn off my phone for instance using the traditional power + volume).  Even some normal apps like Gmail will not open now.

Not sure what to do next...obviously this JB wasn't quite ready for primetime on 12.4 A12 devices after all.  Appreciate any help or ideas to try!
